docs/source/refs.bib

```diff
@@ -38,6 +38,14 @@ @misc{kalchbrenner2018efficient
       archivePrefix={arXiv},
       primaryClass={cs.SD}
 }
+@inproceedings{shen2018natural,
+      title={Natural TTS Synthesis by Conditioning WaveNet on Mel Spectrogram Predictions},
+      author={Shen, Jonathan and Pang, Ruoming and Weiss, Ron J and Schuster, Mike and Jaitly, Navdeep and Yang, Zongheng and Chen, Zhifeng and Zhang, Yu and Wang, Yuxuan and Skerrv-Ryan, Rj and others},
+      year={2017},
+      eprint={1712.05884},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
```

Comment:
Looks like `docs/source/models.rst` is not updated.
But I cannot tell if `Tacotoron2` class is public or private.
If it's public, it has to be added to `models.rst`.

Reply:
We plan to make it public after we include the training pipeline in the examples. If this is the case, should I remove this for now?